docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,128 @@
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/mvk1axfn7e?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Set up a test
+
+Since project tests share the same canvas as Models, you can set them up the same way you would develop and run a Pipeline.
+
+### Develop a test
+
+To develop a project test, start by opening a project:
+
+1. Under the Project Browser, click **Add Test**.
+
+   ![Add a new test](img/project-test-add-new.png)
+
+2. Enter your **Test Name**. Project tests are saved to a tests SQL file by default.
+
+3. Click **OK** to create your new test.
+
+4. Create your test Model by dragging and dropping Gems to your Visual canvas. Connect them to your Data Test. You can also write your test on the Code view. You can use as many Models and other database objects as you’d like in your test.
+
+   ![Project test canvas](img/project-test-canvas.png)
+
+### Run a test
+
+After you’ve developed your project test and executed all of your Models, you can run the test.
+
+1. Click the **Play** button on either the canvas or an individual Gem to execute your test. The outputted table from the Gem is what’s tested.
+   1. Run a whole test. Clicking the canvas **Play** button executes the complete test and shows the interim data after the Data Test Gem. Additionally, it displays the final test status, succeeded, warning, or failed, in a summary.
+   2. Run a partial test. Clicking the Gem **Play** button executes the test SQL only for a particular Gem, and displays that Gem’s data on the output. This option doesn’t execute the data test.
+2. Click **See Run Details** to view the test Summary. Depending on the outcome of the test, the icon displays a different color.
+
+   ![See Run Details](img/project-test-run-details.png)
+
+   You can click to expand the test logs in the Summary to view the dbt logs.
+
+   ![View test summary](img/project-test-summary.png)
+
+In addition to the previous relational integrity test, you can create tests to check that the total payment amount from customers is positive or simply check that all of your tables are still accessible. You can test any series of transformation Gems, because a project test simply asserts that the resulting table meets a certain criteria.
+
+## Fix a failed test
+
+If your project test fails, check your test Model for any changes.
+
+![Failed test](img/project-test-fail.png)
+
```

Comment:
I feel that the happy path should be shown here, since the error "none.get" is really hard to understand. The happy path error can be obtained by following Maciej's video demo to adjust the rows in the tested table.

Reply:
Ack.

Opted instead for a single step for viewing the test output.

Reply:
In this case, I suspect "none.get" is appearing because there is a backend issue unrelated to the test failure. I re-ran the referential integrity test I had created [here](https://beta.prophecy.io/metadata/sql/276?entity=singulardatatest&name=ref_integrity_for_orders) with either zero or one row. When there is more than one row, the test fails: 

<img width="1440" alt="Failure" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/20563764/d99735b2-88f1-4769-ad0e-58fe05df4a3c">
 
Please instead use the above png showing the happy path. If you want to explore the "none.get" error we can file a bug with reproducible steps for Engineering.

Reply:
I'm having trouble getting this screenshot. For now, I'm going to remove a couple of screenshots, including the failed test one, that I don't think users need to see in order to orient themselves in the product. I already thought there were too many on the page. We can add more later if you think they are beneficial.